sources/packages/api/src/route-controllers/restriction/models/restriction.dto.ts

```diff
@@ -33,20 +35,37 @@ export interface RestrictionDetailDTO extends RestrictionSummaryDTO {
 /**
  * DTO to resolve restriction to a student/institution.
  */
-export interface ResolveRestrictionDTO {
+export class ResolveRestrictionAPIInDTO {
+  @IsOptional()
   noteDescription: string;
 }
 
 /**
  * DTO to add restriction to a student/institution.
  */
-export interface AssignRestrictionDTO extends ResolveRestrictionDTO {
+export class AssignRestrictionAPIInDTO extends ResolveRestrictionAPIInDTO {
+  @IsPositive()
```

Comment:
I think it should be  `@Min(0)`, instead of `@IsPositive()` because 0 is also a positive number

Reply:
![image](https://user-images.githubusercontent.com/61259237/184995498-bb939351-0d01-45f7-acef-2a931683adc7.png)